mapie/_compatibility.py

```diff
@@ -0,0 +1,33 @@
+from typing import Any
+
+import numpy as np
+from packaging.version import parse as parse_version
+
+from ._typing import ArrayLike, NDArray
+
+
+def np_quantile_version_below_122(
+    a: ArrayLike,
+    q: ArrayLike,
+    method: str = "linear",
+    **kwargs: Any
+) -> NDArray:
+    """Wrapper of np.quantile function for numpy version < 1.22."""
+    return np.quantile(a, q, interpolation=method, **kwargs)  # type: ignore
+
+
+def np_quantile_version_above_122(
+    a: ArrayLike,
+    q: ArrayLike,
+    method: str = "linear",
+    **kwargs: Any
+) -> NDArray:
+    """Wrapper of np.quantile function for numpy version >= 1.22."""
```

Comment:
No description of the input and output argument arguments ?

Reply:
Initially I did copy-pasted the docstring of numpy and then it felt awkward to do this two times. I hope the docstring is explicit enough